Allow screen readers to read the no results message in the attribute filter

diff --git a/view/frontend/templates/layer/filter/attribute.phtml b/view/frontend/templates/layer/filter/attribute.phtml
index f04140e..0c3c7ba 100644
--- a/src/module-elasticsuite-catalog/view/frontend/templates/layer/filter/attribute.phtml
+++ b/src/module-elasticsuite-catalog/view/frontend/templates/layer/filter/attribute.phtml
@@ -22,7 +22,9 @@
   $datascope = $this->getFilter()->getRequestVar() . 'Filter';
 ?>
 <?php if (!empty($filterItems)) : ?>
-<div data-bind="scope: '<?php echo /* @escapeNotVerified */ $datascope; ?>'">
+<?php // BEGIN PATCH ?>
+<div data-bind="scope: '<?php echo /* @escapeNotVerified */ $datascope; ?>'" aria-live="polite" aria-atomic="true">
+<?php // END PATCH ?>
     <!-- ko template: getTemplate() --> <!-- /ko -->
 </div>

diff --git a/view/frontend/web/template/attribute-filter.html b/view/frontend/web/template/attribute-filter.html
index 5a93e54..a4a111b 100644
--- a/src/module-elasticsuite-catalog/view/frontend/web/template/attribute-filter.html
+++ b/src/module-elasticsuite-catalog/view/frontend/web/template/attribute-filter.html
@@ -26,7 +26,9 @@
 </ol>

 <div class="no-results-message empty" data-bind="if: fulltextSearch() && !hasSearchResult()">
-    <p data-bind="html: getSearchResultMessage()"></p>
+    <!-- BEGIN PATCH -->
+    <p data-bind="html: getSearchResultMessage()" aria-atomic="true"></p>
+    <!-- END PATCH -->
 </div>

 <div class="actions" data-bind="visible: enableExpansion()">
